CHANGELOG.md

```diff
@@ -26,6 +26,14 @@
   [David Airapetyan](https://github.com/davidair)
   [#5184](https://github.com/CocoaPods/CocoaPods/issues/5184)
 
+* Fix `search` command crash while specs directory not exist.
```

Comment:
I also get a crash if the specs directory exists, but has no podspecs within it.  Does this PR handle that case?  Looking at the new unit tests, it doesn't seem to.

Reply:
It's same issue? I don't think so.

Reply:
According to you description, I think it occur in other scene.

Reply:
This needs to move up to the current version